src/Resources/doc/menu_convention.rst

```diff
@@ -0,0 +1,110 @@
+Convention-based menus
```

Comment:
is there additional value in keeping this method? if not, i would deprecate the mode and drop it in the next major version. i feel we have too many similar ways to achieve the same with this bundle.

Reply:
I'm totally in favor of deprecating this provider (especially as it is not compatible with Flex), but this does not belong to this PR